### DISCUSSION TITLE
Question about generating the index file

### DISCUSSION THREAD

**Discussion by sebastianhamza** [KosmosisDire/obsidian-webpage-export#539] [Q&A]
I was looking at the home page (index.html) for the documentation and I was wondering if there is any possibility of automatically generating something like that (or simpler). It's that just manually created? 

I suppose that a basic template can be created and based on some content in the vault, it will generate a basic index.html file as a start.

I am investigating the best ways to publish an obsidian vault as a website and this plugin seems to be the most straightforward (except that index file :) ) Thank you!

**Comment by KosmosisDire** [comment]
If you mean the card view on the docs, there is already a thread about this:
https://github.com/KosmosisDire/obsidian-webpage-export/discussions/379

(If you want that card view it is not straight forward. If you just mean creating any index file in general that is very simple, just create a file in your vault named index)

**Reply by sebastianhamza** [reply]
Thank you! Will give the card view script a try.
